client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkUtils.java

```diff
@@ -92,6 +101,25 @@ public static String appUniqueId(SparkContext context) {
         .getOrElse(context::applicationId);
   }
 
+  public static int celebornShuffleId(ShuffleClient client, CelebornShuffleHandle<?, ? ,?> handle, TaskContext context, Boolean isWriter) {
```

Comment:
```suggestion
  public static int shuffleId(ShuffleClient client, CelebornShuffleHandle<?, ? ,?> handle, TaskContext context, Boolean isWriter) {
```
IMO, no need to emphasize that the return value is celeborn shuffle id.

Reply:
We need to distinguish from appShuffleId and celebornShuffleId, as this PR introduces a mapping from spark's shuffleId to celeborn's shuffleId.